iphone/Classes/KrollBridge.m

```diff
@@ -818,7 +818,8 @@ - (TiModule *)loadTopLevelNativeModule:(TiModule *)module withPath:(NSString *)p
 	}
 
 	NSString* contents = [[[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding] autorelease];
-	KrollWrapper* wrapper = (id) [self loadJavascriptText:contents fromFile:path withContext:kroll];
+	NSURL *url_ = [TiHost resourceBasedURL:path baseURL:NULL];
+	KrollWrapper *wrapper = (id) [self loadCommonJSModule:contents withSourceURL:url_];
```

Comment:
`loadJavascriptText` does some things that `loadCommonJSModule` doesn't, like checking [previously cached modules](https://github.com/appcelerator/titanium_mobile/blob/master/iphone/Classes/KrollBridge.m#L970) and [adding them to the cache](https://github.com/appcelerator/titanium_mobile/blob/master/iphone/Classes/KrollBridge.m#L1001). I am curious that this could lead to other issues regarding module caching.